auto-instrumentation/okhttp/okhttp-3.0/README.md

```diff
@@ -1,7 +1,5 @@
 # Android Instrumentation for OkHttp version 3.0 and higher
 
-# 🚧Work in progress 🚧
```

Comment:
Do you think it might also make sense to put a note about stability? Like experimental?